src/main/java/Imports.java

```diff
@@ -1,6 +1,3 @@
-import java.lang.Byte;
-import java.lang.NumberFormatException;
-
```

Comment:
I believe these should remain, from the filename I would assume the intent was to show the order of imports?

Reply:
> I believe these should remain, from the filename I would assume the intent was to show the order of imports?

Checkstyle flags those as unnecessary imports.   And since they are unnecessary, I removed them.  We could add some meaningful imports if you think it is needed.

Reply:
A file which by its name is supposed to demonstrate the desired import order needs to have some imports from the relevant packages to fulfill its purpose. Our codestyle prescribes java.* to go first, so instead of the pointless java.lang.* imports we can just add java.util.*.

Reply:
> Our codestyle prescribes java.* to go first, so instead of the pointless java.lang.* imports we can just add java.util.* examples here.

Added in 34eade2c0ee92b6495de7c9e2b06a4d231a913c5